README.md

```diff
@@ -64,6 +64,12 @@ to the controller-gen CLI page in the [kubebuilder documentation](https://book.k
 
 ### Running the operator in kind
 
+#### Automated
+
+Run `./hack/kind/setup.sh` to create a kind cluster that has all the
```

Comment:
Correct me if I'm wrong but running this will not actually deploy the operator correct? If so I think we should clarify in the readme what instructions are to set up a cluster and what are to run the operator. WDYT of this structure: 
1. Setting up a dev cluster to run OO
     - Automated
     - Manual
2. Running OO on a cluster
     - Catalog install + OLM install
     - Run local
     - Run with local registry

Reply:
> running this will not actually deploy the operator correct?

That's right, the intention of the setup is to not run the operator but to setup the bare minimal cluster so that operator can be run. It in fact does not even apply the `deploy/common/kubernetes` and `crds` because often you may want to test without these dependencies.

Reply:
> Running OO on a cluster

I like the idea!
How about in a follow up patch so that this one is minimal and only introduces one change - setup a cluster.

Reply:
> I actually feel we should abandon the manual part and rely on an automated setup so that everyone has a similar environment.

100% agree, the more consistent we make environments the better so I'm super down for this 👍 

>  How about in a follow up patch so that this one is minimal and only introduces one change - setup a cluster.

Sounds good ✅